checklist.py

```diff
@@ -450,5 +450,31 @@ def _parse_file(self, fp, root):
         return skbio_imports
 
 
+class FluxCapacitorValidator(RepoValidator):
+    """Ensure that the __future__ statements are fluxing correctly"""
+    reason = ("These files do not have the following import at the start:\n\n"
+              "from __future__ import absolute_import, division,"
+              " print_function\n")
+
+    def _validate(self, root, dirs, files):
+        failures = []
+        expected = set(("absolute_import", "division", "print_function"))
+        for file in files:
+            if file.endswith(".py"):
```

Comment:
`ast` barfs on Cython code, no can do :(

I guess I could write a custom parser for Cython code... but I won't be able to add that to this PR. I can create an issue if you'd like.